### ISSUE TITLE
dask to_datetime can't be read by spark

### ISSUE THREAD

**Issue by birdsarah** [dask/dask#5342]
* make dataframe
* cast one column to datetime with `dd.to_datetime`
* save to parquet using fastparquet and snappy

Try and read that dataframe from spark

```
Py4JJavaError: An error occurred while calling o120.parquet.
: org.apache.spark.SparkException: Job aborted due to stage failure: Task 0 in stage 0.0 failed 4 times, most recent failure: Lost task 0.3 in stage 0.0 (TID 3, ip-172-31-31-113.us-west-2.compute.internal, executor 1): org.apache.spark.sql.AnalysisException: Illegal Parquet type: INT64 (TIMESTAMP_MICROS);
```

**Comment by TomAugspurger** [comment]
Looks like https://github.com/dask/fastparquet/issues/93, http://mail-archives.apache.org/mod_mbox/arrow-dev/201709.mbox/%3CCAJPUwMBtmo4Bt3N-h9PyCyyHdhm3qvO61SSTP65hv6Nm03Fhbg@mail.gmail.com%3E, 

Does passing `times="int96"` when writing work? I'm not sure what the preferred way of doing things going forward is. https://stackoverflow.com/questions/42628287/sparks-int96-time-type may be informative.

**Comment by birdsarah** [comment]
You are my hero @TomAugspurger. I had seen the fastparquet issue but couldn't make it work in my bleary eyed state. I did search a decent amount. I did end up here: https://fastparquet.readthedocs.io/en/latest/details.html#spark-timestamps but didn't put it together. I am trying to think if there's a place to document this that I would have figured it out. There's a docs challenge when you go from the misc kwargs to the parquet backends.

Maybe there's nothing to do here. Thanks so much for the quick response.
